README.md

```diff
@@ -59,6 +59,10 @@ Where:
 |`$LATEST_MILESTONE`|[![](https://img.shields.io/badge/dynamic/xml.svg?label=&color=blue&query=%2F%2Fmetadata%2Fversioning%2Flatest&url=https%3A%2F%2Frepo.spring.io%2Fmilestone%2Fio%2Fprojectreactor%2Ftools%2Fblockhound%2Fmaven-metadata.xml)](https://repo.spring.io/milestone/io/projectreactor/tools/blockhound/)|
 |`$LATEST_SNAPSHOT`|[![](https://img.shields.io/badge/dynamic/xml.svg?label=&color=orange&query=%2F%2Fmetadata%2Fversioning%2Flatest&url=https%3A%2F%2Frepo.spring.io%2Fsnapshot%2Fio%2Fprojectreactor%2Ftools%2Fblockhound%2Fmaven-metadata.xml)](https://repo.spring.io/snapshot/io/projectreactor/tools/blockhound/)|
 
+## Supported Libraries
+1. [Project Reactor](https://projectreactor.io)
```

Comment:
Once we rename to "built-in integrations", we should say "Project Reactor 3.2.x", and mention that starting with Reactor 3.3.0, it comes with built-in support that uses [the SPI](https://github.com/reactor/BlockHound/blob/master/docs/custom_integrations.md)